pcdsdevices/areadetector/detectors.py

```diff
@@ -63,6 +64,28 @@ def get_plugin_graph_edges(self, *, use_names=True, include_cam=False):
                           for src, dest in port_edges]
         return port_edges
 
+    def screen(self, main=False):
+        """
+        Open camViewer screen for camera.
+
+        Parameters
+        ----------
+        main : bool, optional
+            Set to True to bring up 'main' edm config screen.
+            Defaults to False, which opens python viewer.
+        """
+        if shutil.which('camViewer'):
```

Comment:
It's a good thing to check for `camViewer` in the path 👍 

I like how you did the "return early if not found" when attempting to run typhos in `pcdsdevices.interface` - might be worth replicating that here, too.

Reply:
I suspect I was investigating a different way of opening cameras if `camViewer` didn't exist.  (maybe searching for the `{P}:LAUNCH_EDM` PV?). A good catch nonetheless, `camViewer` does just that.